tests/conftest.py

```diff
@@ -3,17 +3,18 @@
 
 from app import create_app
 
-@pytest.yield_fixture(scope='session')
+@pytest.fixture(scope='session')
```

Comment:
@matevskial just out of curiosity, why you replaced the `yield_fixture` with `fixture`?

Reply:
I replaced it because it is deprecated, and pytest gives the following warning:
```shell
PytestDeprecationWarning: @pytest.yield_fixture is deprecated.
  Use @pytest.fixture instead; they are the same.
    @pytest.yield_fixture(scope='session')
```